dotnet/src/SemanticKernel.Abstractions/AI/TextToImage/TextToImageServiceExtensions.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.SemanticKernel.TextToImage;
+
+/// <summary>
+/// Extension methods for <see cref="ITextToImageService"/>.
+/// </summary>
+public static class TextToImageServiceExtensions
+{
+    /// <summary>
+    /// Generate an image matching the given description
```

Comment:
`\\\ Given a prompt and/or an input image, the model will generate a new image.`